### PR TITLE
Update management_commands.md to document that purge_revisions won't …

### PR DESCRIPTION
**Pull request by MeghanaNalla**
…delete revisions that are protected

Fixes #...
Updated the document that purge_revisions won't delete revisions that are protected

_Please check the following:_

-   [ ] Do the tests still pass?[^1]
-   [x] Does the code comply with the style guide?
    -   [ ] Run `make lint` from the Wagtail root.
-   [x] For Python changes: Have you added tests to cover the new/fixed behaviour?
-   [x] For front-end changes: Did you test on all of Wagtail’s supported environments?[^2]
    -   [ ] **Please list the exact browser and operating system versions you tested**:
    -   [ ] **Please list which assistive technologies [^3] you tested**:
-   [x] For new features: Has the documentation been updated accordingly?

**Please describe additional details for testing this change**.

[^1]: [Development Testing](https://docs.wagtail.org/en/latest/contributing/developing.html#testing)
[^2]: [Browser and device support](https://docs.wagtail.org/en/latest/contributing/developing.html#browser-and-device-support)
[^3]: [Accessibility Target](https://docs.wagtail.org/en/latest/contributing/developing.html#accessibility-targets)

### CODE REVIEW

**Comment by lb-** [comment]
Closing as looks like a duplicate of https://github.com/wagtail/wagtail/pull/10761

**Comment by lb-** [comment]
Reopening, I think this is a different thing. Sorry, the description could probably have a bit more info for us to understand what's happening and how this relates to the other PRs you have opened.

File: docs/reference/management_commands.md [hunk 1]
```diff
@@ -54,6 +54,8 @@ days will be deleted.
 
 If the `pages` argument is supplied, only revisions of page models will be deleted. If the `non-pages` argument is supplied, only revisions of non-page models will be deleted. If both or neither arguments are supplied, revisions of all models will be deleted.
 
+purge_revisions won't delete revisions that are protected.
+
```
Comment (lb-) [inline docs/reference/management_commands.md:58]: May I suggest integrating this into the paragraph above. 

```md
This command deletes old revisions which are not protected, in moderation, live, approved to go live, or the latest revision.

If the `days` argument is supplied, only revisions older than the specified number of days will be deleted.

If the `pages` argument is supplied, only revisions of page models will be deleted. If the `non-pages` argument is supplied, only revisions of non-page models will be deleted. If both or neither arguments are supplied, revisions of all models will be deleted.
```

**Comment by lb-** [comment]
Upon further inspection, this appears to be the docs for this PR. 

https://github.com/wagtail/wagtail/pull/10759

@MeghanaNalla could you close this PR and add the docs to the branch in that other PR.

I assume this PR shouldn't be merged on its own?

**Comment by MeghanaNalla** [comment]
@lb- yaa sure

**Pull request closed without merging**